pkg/api/store/workload/workload_store.go

```diff
@@ -96,6 +96,8 @@ func setScheduling(apiContext *types.APIContext, data map[string]interface{}) {
 		state := getState(data)
 		state[getKey(nodeName)] = nodeID
 		setState(data, state)
+	} else {
+		values.PutValue(data, "", "nodeId")
```

Comment:
@StrongMonkey could you explain why we need to propagate this parameter as empty?

Reply:
@alena1108 if user unset scheduling rule, we need to update nodeId to null. The reason we need to set it empty is because backend is adopting merge update strategy so we need to specifically set it empty.